### ISSUE TITLE
Is there any requirement for the brand parameter in this file（vehicle_param.pb.txt）? The brand of my own car will report an error.

### ISSUE THREAD

**Issue by lq666666666** [ApolloAuto/apollo#15453]
I defined my own car brand in calibration/data. After build, an error will be reported when starting dreamview, as shown below. When I changed the brand parameter to LINCOLN MKZ, it can be started successfully, but the brand parameter is required
![dreamiew啟動失敗原因](https://github.com/ApolloAuto/apollo/assets/86345770/dd77c62c-22de-4438-98e8-a119b923a03e)

**Comment by dajiba6** [comment]
have you defined the type in vehicle_config.proto?

**Comment by lq666666666** [comment]
No, is it only necessary to add my own brand in VehicleBrand and is there any configuration description? In addition, I would also like to ask if there is any document related to the calibration content of my car?

**Comment by daohu527** [comment]
Yes, if you want to add new vehicle `brand` is one of the fields. You also need to modify other vehicle parameters, such as vehicle length, width, turning radius, etc.

After that, sensor calibration is required to obtain the position relationship of the sensors.

https://github.com/ApolloAuto/apollo/blob/6edbb1f8fcee5ad2e75109b89de799236971548e/modules/calibration/data/kitti_140/vehicle_param.pb.txt#L1-L3
